docs/learning/SupportedPlatforms.md

```diff
@@ -4,7 +4,7 @@
 
 iTwin.js **backends** are built and tested on the following:
 
-- Debian 9 "Stretch" and Debian 10 "Buster"
+- Debian 10 "Buster" and Debian 11 "Bullseye"
```

Comment:
We should keep Debian 9 on the list, it is still technically supported due to the glibc version.

Reply:
i removed debian 9 because its already reached [EOL](https://wiki.debian.org/LTS) and wasnt sure we want to explcitly mentioning supporting it. would mentioning just the min glibc ver suffice so we dont have to add debian 9 to the list here?

Reply:
Yea, that's fair enough. Are you going to add the min glibc as well this release?

Reply:
Yea ill add it to this pr, if someone tells me what the min glibc ver is

Reply:
Our minimum is `2.24` based off of the Debian 9 version.

Reply:
looks like we already list the min glibc ver: [itwinjs.org/learning/supportedplatforms/#backend-prerequisites](https://www.itwinjs.org/learning/supportedplatforms/#backend-prerequisites), see L30